R/mod_covid_adjustment_server.R

```diff
@@ -1,7 +1,7 @@
 #' covid_adjustment Server Functions
 #'
 #' @noRd
-mod_covid_adjustment_server <- function(id, params) {
+mod_covid_adjustment_server <- function(id, covid_adjustment_data, params) {
```

Comment:
```suggestion
mod_covid_adjustment_server <- function(id, covid_adjustment_data(), params) {
```

Would this also need to be the reactive?